packages/utils/logger/lib/output-file-configuration.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { transports, format } = require('winston');
+const { LEVEL_LABEL, LEVELS } = require('./constants');
+const { prettyPrint } = require('./formats');
+
+const getPlainText = format.printf(({ message }) => {
+  return message.replace(
+    // eslint-disable-next-line no-control-regex
+    /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g,
```

Comment:
Could you add a comment to explain what this is used for?
I believe this is for removing chalk color codes, but it would be nice to have a small explanation